### PR TITLE
7 feature 4b money pool

### PR DESCRIPTION
**Pull request by Andy-Guo-21**
Using PlayerInterface for now; Player class will need to implement

### CODE REVIEW

File: src/main/java/Pool.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    private final PlayerInterface[] players;
+    private int[] bets;
+
+    /**
+     * Class constructor for Pool
+     *
+     * @param players   the players who are playing this game
+     */
+    public Pool(PlayerInterface[] players) {
+        this.players = players;
```
Comment (enfz) [inline src/main/java/Pool.java:16]: Might need a factory for the player entities to have clean architecture

File: src/main/java/PlayerInterface.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+public interface PlayerInterface {
+
+    void addMoney(int amount);
```
Comment (enfz) [inline src/main/java/PlayerInterface.java:3]: Player interface might need extra things such as the cards in the players hand

**Review by enfz — APPROVED** [review]
Implementation looks good might need to make a future issue to fix any clean architecture required

**Review by Jefferson-liu — APPROVED** [review]
We will need factories and interactors to comply with clean architecture, but the core logic looks fine

**Pull request merged**